src/routes/movies.tsx

```diff
@@ -0,0 +1,139 @@
+import '../elements/emby-scroller/emby-scroller';
```

Comment:
This should probably be moved to `src/routes/movies/index.tsx` since the directory exists now.

Reply:
https://github.com/jellyfin/jellyfin-web/pull/3792/commits/0914f9b72fcb82d488f62aed3650d7427c0136fe